de.metas.acct.base/src/main/java/de/metas/acct/process/Documents_FactAcct_Creation_For_Posted.java

```diff
@@ -0,0 +1,92 @@
+package de.metas.acct.process;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+import org.adempiere.acct.api.IPostingRequestBuilder.PostImmediate;
+import org.adempiere.acct.api.IPostingService;
+import org.adempiere.util.ILoggable;
+import org.adempiere.util.Services;
+import org.adempiere.util.lang.ITableRecordReference;
+import org.adempiere.util.lang.impl.TableRecordReference;
+import org.compiere.process.SvrProcess;
+import org.compiere.util.TimeUtil;
+
+import de.metas.acct.api.IDocumentBL;
+
+/*
+ * #%L
+ * de.metas.acct.base
+ * %%
+ * Copyright (C) 2016 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+/**
+ * The documents (created one day before) that were marked as posted but have no fact accounts will be reposted by this process
+ * 
+ * @author metas-dev <dev@metasfresh.com>
+ *
+ */
+public class Documents_FactAcct_Creation_For_Posted extends SvrProcess
+{
+	final ILoggable loggable = ILoggable.THREADLOCAL.getLoggable();
+
+	@Override
+	protected String doIt() throws Exception
+	{
+		// this process is posting documents that were created one day before the process runs
+		final Timestamp startTime = TimeUtil.getPrevDay(new Timestamp(System.currentTimeMillis()));
+
+		// list all the documents that are marked as posted but have no fact accounts.
+		// this list will not include the documents with no fact accounts that were not supposed to be posted (always 0 in posting)
+		final List<Object> documentsPostedNoFacts = Services.get(IDocumentBL.class).retrievePostedWithoutFactActt(getCtx(), startTime);
+
+		if (documentsPostedNoFacts.isEmpty())
+		{
+			// do nothing
+			return "All documents are posted";
+		}
+
+		final ILoggable loggable = ILoggable.THREADLOCAL.getLoggable();
+
+		final IPostingService postingService = Services.get(IPostingService.class);
+
+		for (final Object document : documentsPostedNoFacts)
+		{
+			final ITableRecordReference tableRecordRef = TableRecordReference.of(document);
+
+			// Note: Do not change this message!
+			// The view de_metas_acct.Reposted_Documents is based on it.
+			loggable.addLog("Document Reposted: AD_Table_ID = {}, Record_ID = {}.", tableRecordRef.getAD_Table_ID(), tableRecordRef.getRecord_ID());
```

Comment:
Please also output the documents table name and documentn number, if it is a docaction.
Also check out my remarks at the view's DDL